test/extensions/filters/http/ext_authz/ext_authz_test.cc

```diff
@@ -53,6 +53,8 @@ namespace HttpFilters {
 namespace ExtAuthz {
 namespace {
 
+constexpr char filter_config_name[] = "ext_authz_filter";
```

Comment:
nit: In the Envoy codebase we use ConstantVar or CONSTANT_VAR

https://github.com/envoyproxy/envoy/blob/main/STYLE.md